server/sandbox_run_linux.go

```diff
@@ -427,8 +427,8 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	for idx := range cleanupFuncs {
 		defer func(currentFunc int) {
 			if err != nil {
-				if err2 := cleanupFuncs[currentFunc](); err != nil {
-					log.Debugf(ctx, err2.Error())
+				if err := cleanupFuncs[currentFunc](); err != nil {
```

Comment:
I feel like I'd prefer the explicit err2 here. Matches the rest of the code, and prevents confusion that shadowing causes

Reply:
Okay, changed it as suggested :+1: